tools/cythonize.py

```diff
@@ -62,7 +62,7 @@ def process_pyx(fromfile, tofile):
         # The `cython` command need not point to the version installed in the
         # Python running this script, so raise an error to avoid the chance of
         # using the wrong version of Cython.
-        raise OSError('Cython needs to be installed in Python as a module')
+        raise OSError('Cython needs to be installed in Python as a module') from None
```

Comment:
Both the `except ImportError` blocks in this PR should use `from e` instead of `from None`, as `ImportError`s can often contain useful debugging info.

Reply:
Line is too long, should be < 80 characters. Try 
```
msg = 'Cython needs to be installed in Python as a module'
raise OSError(msg) from e
```
Note you will need `except ImportError as e:` as Eric suggested.

Reply:
Thanks - I've updated the lines based on your suggestions.